certbot_dns_duckdns/cert/client.py

```diff
@@ -78,5 +92,6 @@ def _get_duckdns_client(self) -> DuckDNSClient:
 
         :return: the created DuckDNSClient object
         """
-
-        return DuckDNSClient(self.conf("token"))
+        token = self.conf("token") or self.credentials.conf("token")
+        print("Token:",token)
```

Comment:
The token should not be print to stdout in the release version, to prevent unintended access to the API token if this is written to a log file for example

Reply:
Oh no, I forgot to remove this after testing. Sorry about that!